[DOCS] Adds 9.4.3 Kibana release notes (#274082)
elastic/kibana: docs/release-notes/index.md

% FEATURES, ENHANCEMENTS, FIXES
% Paste in index.md

## 9.4.3 [kibana-9.4.3-release-notes]

### Features and enhancements [kibana-9.4.3-features-enhancements]

**Elastic Observability solution**:

> For the Elastic Observability 9.4.3 release information, refer to [Elastic Observability Solution Release Notes](docs-content://release-notes/elastic-observability/index.md).

Reviewer: not worth having this section just to link away IMO, this link already exists in fixes, I think it should be sufficient
```suggestion
```

* Includes input packages in the Integrations **Signal type** filter [#272585]({{kib-pull}}272585).
* Fixes security rule names not displaying in the integration **Assets** accordion [#272089]({{kib-pull}}272089).
* Fixes the agent logging level dropdown not updating after clicking **Reset to policy** on the agent **Settings** tab [#271964]({{kib-pull}}271964).
* Fixes broken asset references in the integration **Assets** tab when an integration is reinstalled or upgraded from a different {{kib}} space [#271800]({{kib-pull}}271800).
* Fixes per-integration output overrides for OpenTelemetry (OTel) integrations, and now raises a validation error when an unsupported output type is assigned instead of failing silently [#270487]({{kib-pull}}270487).

**Discover**:
* Fixes `null` and `undefined` field values in the {{esql}} document summary column incorrectly counting toward the `discover:maxDocFieldsDisplayed` limit, which caused fields with real values to be hidden [#273610]({{kib-pull}}273610).
* Fixes the **Extend** action icon in the Background searches panel being invisible in dark mode [#272754]({{kib-pull}}272754).

> * Fixes Discover session panels in dashboards failing to load after **Copy to Spaces** or import when the target space already contains remapped sessions [#272610]({{kib-pull}}272610).

Reviewer: I would move this to dashboards and viz because it's specifically about panels

Author: Moved

**{{esql}} editor**:
* Fixes a regression where KQL autocomplete suggestions stopped working in the {{esql}} editor [#273931]({{kib-pull}}273931).

> * Fixes editing of object fields in the {{esql}} **Index** editor when using dot-notation column names [#273303]({{kib-pull}}273303).

Reviewer: ```suggestion
* Fixes editing of object fields in the {{esql}} lookup index editor when using dot-notation column names [#273303]({{kib-pull}}273303).
```
that editor is only for lookup indices so correcting the name (that's how we generally refer to it)

**Elastic Observability solution**:
For the Elastic Observability 9.4.3 release information, refer to [Elastic Observability Solution Release Notes](docs-content://release-notes/elastic-observability/index.md).

**Elastic Security solution**:
For the Elastic Security 9.4.3 release information, refer to [Elastic Security Solution Release Notes](docs-content://release-notes/elastic-security/index.md).

**Kibana platform**:
* Adds `notifications.connectors.default.email` as a Docker environment variable [#272761]({{kib-pull}}272761).

> 
> **Kibana security**:
> * Fixes user profile retrieval in reverse proxy `run_as` configurations, ensuring the nav bar displays the effective user's avatar and name rather than the proxy user's [#271314]({{kib-pull}}271314).

Reviewer: ```suggestion
**Kibana platform**:
* Adds `notifications.connectors.default.email` as a Docker environment variable [#272761]({{kib-pull}}272761).
* Fixes user profile retrieval in reverse proxy `run_as` configurations, ensuring the nav bar displays the effective user's avatar and name rather than the proxy user's [#271314]({{kib-pull}}271314).
```
combine

**Machine Learning**:

> * Fixes product documentation installation timing out when the ML inference endpoint scales up from zero replicas (cold-start), and improves error messages when installation fails [#270005]({{kib-pull}}270005).

Reviewer: That's more general to AI set up in Kibana/Gen AI settings than just ML

Author: Adding this to the "Kibana platform" section.

**Management**:
* Logs Dev Tools Console requests through the `elasticsearch.query` debug logger (method, path, and response status; request bodies are not included since Console streams them), and adds support for URL path prefixes in `elasticsearch.hosts` for Console proxy requests [#271562]({{kib-pull}}271562).

> * Fixes the **Content Connectors** entry in **Stack Management** appearing in the sidebar for users without the required capability [#271709]({{kib-pull}}271709).

Reviewer: align with the other release notes (not in same category + wording)

Author: Created a new connectivity section for this PR and revised summary.
